### ISSUE TITLE
Accessibility: Elements must only use supported ARIA attributes

### ISSUE THREAD

**Issue by bedmison** [web-cat/web-cat-subsystem-Core#8]
Ensure an element's role supports its ARIA attributes. This shows up on most radio buttons and checkboxes, where they have aria-pressed="true" set. This ARIA attribute is not allowed and should be removed.
